app/controllers/Application.scala

```diff
@@ -75,7 +75,14 @@ object Application extends Controller with PanDomainAuthActions {
       sections <-  getSortedSections
       desks <- getSortedDesks
       sectionsInDesks <- getSectionsInDesks
-      commissioningDesks <- TagService.getTags(Config.tagManagerUrl+ "/hyper/tags?limit=100&query=tracking/commissioningdesk/&type=tracking&searchField=path").map(_.getOrElse(List[Tag]()))
+      commissioningDesks <- TagService.getTags(Config.tagManagerUrl+
```

Comment:
It was me that made this a bit messy in the first place, but would it be possible to move the logic here into TagService so that the for comprehension is a bit more readable?